From f6a9d492115b7d57e9e0de8e9e2e89acccbe8857 Mon Sep 17 00:00:00 2001
From: David Redondo <kde@david-redondo.de>
Date: Fri, 22 Jan 2021 16:11:02 +0000
Subject: [PATCH] kde4breeze: Breeze Light is the new default

Otherwise this kconf_update binary writes Breeze colors to the users kdeglobals.
I created a new user, added its ~/.config/kdeglobals to auditd and saw that this binary wrote to it.
BUG: 431917


(cherry picked from commit 9d404df129774922efaa744423597f8af7ca9e32)
---
 misc/kde4breeze/src/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc/kde4breeze/src/main.cpp b/misc/kde4breeze/src/main.cpp
index d4d541b2..af6c9f6b 100644
--- a/misc/kde4breeze/src/main.cpp
+++ b/misc/kde4breeze/src/main.cpp
@@ -88,7 +88,7 @@ void updateKdeGlobals()
     }
 
     const QString widgetStyle = readConfigValue(lnfConfig, defaultLnfConfig, "KDE", "widgetStyle", "Breeze").toString();
-    const QString colorScheme = readConfigValue(lnfConfig, defaultLnfConfig, "General", "ColorScheme", "Breeze").toString();
+    const QString colorScheme = readConfigValue(lnfConfig, defaultLnfConfig, "General", "ColorScheme", "BreezeLight").toString();
     const QString icons = readConfigValue(lnfConfig, defaultLnfConfig, "Icons", "Theme", "breeze").toString();
 
     cloneColorScheme(colorScheme);
-- 
GitLab

